.mvn/wrapper/MavenWrapperDownloader.java

```diff
@@ -106,12 +99,11 @@ protected PasswordAuthentication getPasswordAuthentication() {
             });
         }
         URL website = new URL(urlString);
-        ReadableByteChannel rbc;
-        rbc = Channels.newChannel(website.openStream());
-        FileOutputStream fos = new FileOutputStream(destination);
-        fos.getChannel().transferFrom(rbc, 0, Long.MAX_VALUE);
-        fos.close();
-        rbc.close();
+        try(ReadableByteChannel rbc = Channels.newChannel(website.openStream())){
+            try(FileOutputStream fos = new FileOutputStream(destination)){
+                fos.getChannel().transferFrom(rbc, 0, Long.MAX_VALUE);
+            }
+        }
```

Comment:
This whole file is provided by Maven Wrapper and it will be overwritten when I will upgrade the Wrapper
https://github.com/takari/maven-wrapper

Reply:
Hi

maven-wrapper is deprecated, now is part of maven.
I have never fix Apache software ... may be the first time :)

all the best
matteo baccan

Reply:
Interesting, I didn't know that. I will then remove `.mwn` folder, thanks a lot!